docker-compose-2.yml

```diff
@@ -0,0 +1,107 @@
+version: '2.1'
```

Comment:
This file is for testing with the new version of kumascript while still being able to run the old.

Before deplying, I'd just copy this file over to docker-compose.yml